pyfake/adjective.py

```diff
@@ -1,14 +1,15 @@
 from .generator import Generator
+from pathlib import Path
 import random
 import yaml
 
+with open(str(Path(__file__).parent) + '/config/adjectives.yml', 'r') as file:
```

Comment:
You can do a bit simpler: `open(Path(__file__).parent / "config/adjectives.yml) as file` or even just `yaml.safe_load((Path(__file__).parent / "config/adjectives.yml).read_text())`